IMPLEMENTATION_COVERAGE.md

```diff
@@ -1,426 +1,539 @@
+- [ ] get_query_execution
+- [ ] get_query_results
+- [ ] list_named_queries
+- [ ] list_query_executions
+- [ ] start_query_execution
+- [ ] stop_query_execution
 
-## elb - 34% implemented
-- [ ] add_tags
-- [X] apply_security_groups_to_load_balancer
-- [ ] attach_load_balancer_to_subnets
-- [X] configure_health_check
-- [X] create_app_cookie_stickiness_policy
-- [X] create_lb_cookie_stickiness_policy
-- [X] create_load_balancer
-- [X] create_load_balancer_listeners
-- [ ] create_load_balancer_policy
-- [X] delete_load_balancer
-- [X] delete_load_balancer_listeners
-- [ ] delete_load_balancer_policy
-- [ ] deregister_instances_from_load_balancer
+## autoscaling - 42% implemented
+- [X] attach_instances
+- [X] attach_load_balancer_target_groups
+- [X] attach_load_balancers
+- [ ] complete_lifecycle_action
+- [X] create_auto_scaling_group
+- [X] create_launch_configuration
+- [X] create_or_update_tags
+- [X] delete_auto_scaling_group
+- [X] delete_launch_configuration
+- [ ] delete_lifecycle_hook
+- [ ] delete_notification_configuration
+- [X] delete_policy
+- [ ] delete_scheduled_action
+- [ ] delete_tags
 - [ ] describe_account_limits
-- [ ] describe_instance_health
-- [ ] describe_load_balancer_attributes
-- [ ] describe_load_balancer_policies
-- [ ] describe_load_balancer_policy_types
+- [ ] describe_adjustment_types
+- [X] describe_auto_scaling_groups
+- [X] describe_auto_scaling_instances
+- [ ] describe_auto_scaling_notification_types
+- [X] describe_launch_configurations
+- [ ] describe_lifecycle_hook_types
+- [ ] describe_lifecycle_hooks
+- [X] describe_load_balancer_target_groups
 - [X] describe_load_balancers
+- [ ] describe_metric_collection_types
+- [ ] describe_notification_configurations
+- [X] describe_policies
+- [ ] describe_scaling_activities
+- [ ] describe_scaling_process_types
+- [ ] describe_scheduled_actions
 - [ ] describe_tags
-- [ ] detach_load_balancer_from_subnets
-- [ ] disable_availability_zones_for_load_balancer
-- [ ] enable_availability_zones_for_load_balancer
-- [ ] modify_load_balancer_attributes
-- [ ] register_instances_with_load_balancer
-- [ ] remove_tags
-- [ ] set_load_balancer_listener_ssl_certificate
-- [ ] set_load_balancer_policies_for_backend_server
-- [X] set_load_balancer_policies_of_listener
+- [ ] describe_termination_policy_types
+- [X] detach_instances
+- [X] detach_load_balancer_target_groups
+- [X] detach_load_balancers
+- [ ] disable_metrics_collection
+- [ ] enable_metrics_collection
+- [ ] enter_standby
+- [X] execute_policy
+- [ ] exit_standby
+- [ ] put_lifecycle_hook
+- [ ] put_notification_configuration
+- [ ] put_scaling_policy
+- [ ] put_scheduled_update_group_action
+- [ ] record_lifecycle_action_heartbeat
+- [ ] resume_processes
+- [X] set_desired_capacity
+- [X] set_instance_health
+- [ ] set_instance_protection
+- [ ] suspend_processes
+- [ ] terminate_instance_in_auto_scaling_group
+- [X] update_auto_scaling_group
 
-## dynamodb - 100% implemented (is dynamodbv2)
```

Comment:
Im assuming the script that generates this misses some stuff in dynamo?

Reply:
I suspect it does. This regeneration also introduced alphabetic sorting of service names so the location in the files is moved (and the implementation coverage is now reported at 36%)